mcp/sensors/csbf_dgps.c

```diff
@@ -406,4 +518,181 @@ void * DGPSMonitor(void * arg)
         }
         i_char++;
     }
+    return NULL;
+}
+
+/**
+ * @brief Helper to start GNSS UDP receiver
+ * 
+ * @param ipaddr 
+ * @param port 
+ * @param data struct into which the IP and port are stored
+ * @returns Result of last snprintf call
+ */
+int populateSocketData(char * ipaddr, char * port, struct GPSsocketData *data) {
```

Comment:
A version of this already lives in star_camera_transmit.c/h we could probably find a better place for it to live since its now used other places.